README.md

```diff
@@ -1,9 +1,51 @@
-# [aleph-node](https://alephzero.org/)
+[![LOGO][aleph-logo]][aleph-homepage]
 
-[![unit-tests](https://github.com/Cardinal-Cryptography/aleph-node/actions/workflows/unit_tests.yml/badge.svg)](https://github.com/Cardinal-Cryptography/aleph-node/actions/workflows/unit_tests.yml)
+[![Unit tests][unit-tests-badge]][unit-tests]
+[![E2E Tests][e2e-tests-badge]][e2e-tests]
+[![Apache 2.0 Licensed][license-image]][license-link]
 
-[![integration-tests](https://github.com/Cardinal-Cryptography/aleph-node/actions/workflows/integration_tests.yml/badge.svg)](https://github.com/Cardinal-Cryptography/aleph-node/actions/workflows/integration_tests.yml)
 
-[![deploy](https://github.com/Cardinal-Cryptography/aleph-node/actions/workflows/deploy.yml/badge.svg)](https://github.com/Cardinal-Cryptography/aleph-node/actions/workflows/deploy.yml)
+This repository contains the Rust implementation of [Aleph Zero][aleph-homepage] blockchain node based on the [Substrate][substrate-homepage] framework.
 
-Node implementation for aleph blockchain built with Substrate framework.
+Aleph Zero is an open-source layer 1 blockchain platform focused on privacy, scalability and energy efficiency. It is based on a unique, peer-reviewed consensus algorithm - AlephBFT (as described in our [paper][aleph-bft-paper] and implemented [here][aleph-bft-link]).
+
+Aleph node is based on a Substrate node where the default finality gadget (GRANDPA) has been replaced with AlephBFT. Block authoring is realized with Substrate's Aura. The chain is run in periodic sesssions (900 blocks each) utilizing the Session pallet. The authorities in each session serve for both Aura and AlephBFT authorities, and on top of that are responsible for running the Aggregator protocol producing multisignatures of finalized blocks.
+
+
+### Installation
+
+1. Set up the Rust development environment following the instructions [here][rust-installation]
```

Comment:
Shouldn't we inform about exact version of nightly?

Reply:
We probably should. Which one is that?

Reply:
https://www.notion.so/cardinalcryptography/How-to-setup-a-repository-7e240b93ab14471d98d59df9430375b0